### ISSUE TITLE
Version 3.0.0-rc3 is not production ready

### ISSUE THREAD

**Issue by thesayyn** [ericclemmons/start-server-webpack-plugin#31]
Today I have upgraded my plugin to a version that contains the necessary fixes such as auto restart regarding the "web pack" emit hooks, however, 3.0.0-rc3 contains a bunch of unnecessary console.log console.warn.
So  there's no way to remove these debugging lines and it is so itchy

**Comment by thesayyn** [comment]
I think there should be an option that we can suppress the logs through
Something like `{  verbose: true | false }`

**Comment by wmertens** [comment]
Ok, so default verbose=true, and if it's false it doesn't log anything?

**Comment by thesayyn** [comment]
mean there is no option to suppress logs. it shows everything by default

**Comment by coreyleelarson** [comment]
I agree with @thesayyn. I always try to disable any debug/info logging from libraries. The @beta version will work perfectly for my use case, but it would be nice to clean up the logging. It's not that big of a deal, but it would be nice and pretty simple to implement.

**Comment by wmertens** [comment]
@coreyleelarson 	so you never want to see any message, not when starting, reloading, crashing, nothing?

**Comment by coreyleelarson** [comment]
@wmertens That is correct. I manage my own server logging, and additional library logging muddies up the terminal.

**Comment by wmertens** [comment]
Ok, personally I always want to know if there's an error, so what do you
propose for configuring logging? All/nothing or all/errors-only/nothing?
And is the `sswp> ` prefix ok or should it be different?

**Comment by coreyleelarson** [comment]
All/errors-only/nothing would be great. I don't really have a preference on the prefix.
